src/modules/dashboard/components/ActionsPage/ActionsPage.css

```diff
@@ -5,6 +5,23 @@
   background-color: white;
 }
 
+.recoveryTag {
+  padding: 4.5px 8px;
+  height: 20px;
+  width: 69px;
+  position: absolute;
```

Comment:
This has no reason to be an absolute element.

- Refactor `.container` to not include a top border
- create a `divider` element that has the solely purpose of displaying that diving line _(the border you just removed)_
- place your tag component inside the `.container` element, above the divider